src/components/shared/form/controls.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import styled from '@styled-components';
+
+interface FormInputProps {
+  hasError?: boolean;
+}
+
+const FormLabel = styled.label`
+  font-weight: 800;
+  padding: 0.825em 0;
+`;
+const FormInput = styled.input<FormInputProps>`
+  padding: 0.425em;
+  border-radius: 3px;
+  box-shadow: 0 0 1px gray;
+  border: ${(props) =>
+    props.hasError ? '1px solid red' : '1px solid lightgray;'};
+  background: ${(props) => (props.hasError ? '#fff1f4' : 'white;')};
+`;
+const SelectInput = styled.select<FormSelectInputProps>`
+  padding: 0.425em;
+  border-radius: 3px;
+  box-shadow: 0 0 1px gray;
+  border: 1px solid lightgray;
+  appearance: none;
+  ​ :focus {
+    border-color: black;
+  }
+  background: url(http://cdn1.iconfinder.com/data/icons/cc_mono_icon_set/blacks/16x16/br_down.png)
+    no-repeat 95% 50% white;
+  border: ${(props) =>
+    props.hasError ? '1px solid red' : '1px solid lightgray;'};
+  background: ${(props) => (props.hasError ? '#fff1f4' : 'white;')};
```

Comment:
Consider doing something like
```suggestion
  background: ${({ theme, hasError }) =>
    hasError ? '#fff1f4' : theme.colors.baseinvert};
```
To avoid the use of hardcoded color values. Also, add `#fff1f4` to `constants.ts` and `theme.ts` under `src/styles`

Reply:
i think this issue is now fixed?